README.md

```diff
@@ -129,6 +129,7 @@ The above example uses app-level credentials i.e., publish profile file for depl
 
 Follow the steps to configure the secret:
 
+* **Note**: As of October 2020, Linux web apps will need the app setting `WEBSITE_WEBDEPLOY_USE_SCM` set to `true` before continuing. This requirement will be removed in the future.
```

Comment:
Thanks for adding this @JasonFreeberg. We may also want to add this to individual workflow templates in https://github.com/Azure/actions-workflow-samples/tree/master/AppService for greater visibility of this message.